README.md

```diff
@@ -1,280 +1,231 @@
+The netstats client has to connect to the blockchain node. Therefore, we have to find out the node's IP address with
 
 ```sh
-parity account new --chain trustlines-spec.json -d [path/to/node/foler]
+docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' laika-node
 ```
 
-For the rest of this documentation to become a validator we will assume you ran:
+Now, the netstats client can be started with
 
 ```sh
-parity account new --chain trustlines-spec.json -d validator_node
+$ docker run --name netstats-client --network network-laika -e trustlines/netstats-env -e RPC_HOST=<node IP address> -e RPC_PORT=8545 trustlines/netstats-client:master
 ```
 
-You will be prompted to enter a password twice to protect this private key. You need to remember this password and use it whenever you want to use the private key. After successfully creating an account, you will see displayed the public address corresponding to that account (in format 0x6c4cbad9865dbfda5d5c5e2e353623b07ace71e6), keep that address somewhere.
+#### Monitor
+
+The monitor watches the blockchain and makes sure that validators are online and do not equivocate. Every node in the
+network should run it and users should regularly check for reports of misbehaving validators.
 
-For running a node as a validator, you will need to store your password in a file.
+Assuming the blockchain node was configured as described above, this command will start the monitor:
 
 ```sh
-echo [mypassword] > password.pwd
+$ mkdir -p trustlines/monitor/reports trustlines/monitor/state
+$ docker run -d --name tlbc-monitor --network network-laika -v $(pwd)/trustlines/shared:/config -v $(pwd)/trustlines/monitor/state:/state -v $(pwd)/trustlines/monitor/reports:/reports trustlines/tlbc-monitor -c /config/trustlines-spec.json -r /reports -d /state -u http://<node IP address>:8545
 ```
 
-#### Setup For Users Using Only CLI
+#### Bridge
 
-These are the steps to run a node as a simple user.
+Validators of the Trustlines Blockchain have to run the bridge that converts TLN tokens on the Ethereum chain to TLC
+tokens on the Trustlines Blockchain. Non-validators should not run a bridge node.
 
-Independently of whether you created an account or not, you can start a node to sync with the trustlines chain with the following command:
+The bridge requires an Ethereum mainnet node which can be a light client. To start one, execute
 
 ```sh
-parity --chain trustlines-spec.json -d user_node --auto-update=none --no-download
+$ docker network create network-ethereum
+$ mkdir -p trustlines/databases/goerli
+$ docker run -d --name goerli-node --network network-ethereum -v $(pwd)/trustlines/databases/goerli:/data/database parity/parity:stable --light --no-download --auto-update none --chain goerli --db-path /data/database --base-path /data --no-hardware-wallets --jsonrpc-apis safe --jsonrpc-hosts all --jsonrpc-cors all --jsonrpc-port 8545 --no-ipc --no-secretstore --no-color
```

Comment:
This fails with a permission denied error:

```
Error upgrading parity data: CannotWriteVersionFile(Os { code: 13, kind: PermissionDenied, message: "Permission denied" })
```

If I don't use a volume I get

```
Error upgrading parity data: CannotCreateConfigPath(Os { code: 13, kind: PermissionDenied, message: "Permission denied" })
```

Reply:
Because of the shared volume. Have a look at [this comment](https://github.com/trustlines-protocol/blockchain/blob/0b72bd96d1f7987abfea2b1257743718453d7694/docker/docker-compose.yaml#L35)

Reply:
Format as the review comments above...

Reply:
I've seen this comment, but I'm not sure if this is the issue. That's why I tried not using a volume at all and it still failed.

Also, if it is the issue, how can we fix it? We don't have a compose file here.

Reply:
> That's why I tried not using a volume at all and it still failed.

So just as a standard node without a validator key?

> Also, if it is the issue, how can we fix it? We don't have a compose file here.
1. Compose files can't add options to a container that _Docker_ does not support. So you can just use `docker run --user...` herefore.
2. You can also just use (a) different volume(s) for this node. :man_shrugging:

Reply:
I tried (2), but it doesn't work. I get the same error message as above.

I've added the `--user root` option, but I don't think this is an acceptable solution. Root permissions should not be necessary to run a node.